client/core/trade.go

```diff
@@ -146,6 +146,12 @@ type trackedTrade struct {
 	notify        func(Notification)
 	epochLen      uint64
 	fromAssetID   uint32
+	// redeemFeeSuggestion is cached fee suggestion for redemption. We can't
+	// request a fee suggestion at redeem time because it would require making
+	// the full redemption routine async (TODO?). This fee suggestion is
+	// intentionally not stored as part of the db.OrderMetaData, and should
+	// be repopulated if the client is restarted.
+	redeemFeeSuggestion uint64
```

Comment:
64 bit atomic load/store bug issue here.  Could probably be 32 bit, but perhaps best to put it as the first field in `trackedTrade`.